packages/use-request/package.json

```
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/alibaba/hooks",
   "peerDependencies": {
-    "react": "^16.8.6"
+    "react": "^16.8 || v17.0"
```

It seems should be like this:

```suggestion
    "react": "^16.8 || ^17.0"
```